ouroboros-consensus-cardano/src/unstable-cardano-testlib/Test/Consensus/Cardano/ProtocolInfo.hs

```diff
@@ -321,6 +309,7 @@ mkTestProtocolInfo
               SL.exampleConwayGenesis
           )
           emptyCheckpointsMap
+          protocolVersion -- FIXME: it seems that if the hard fork triggers contain 'TriggerHardForkAtVersion' we will need to use the largest version that appears there.
```

Comment:
If we expect that we will actually transition to that era in the given test, then yes :+1: We might even need to set it to a higher version in case there are intra-era HFs in the last era.

Reply:
We could explain in `protocolInfoCardano` the relation between (this) protocol version and hard fork triggers and refer to that comment in the docs for `mkTestProtocolInfo`. In principle, there is no strict requirement about what the protocol version should be based on the hard fork trigger information. This is up to the user.